README.md

```diff
@@ -2,7 +2,13 @@
 
 # Symbolicator
 
-The symbolicator util symbolicates memory leaks and crash reports.
+The binary code of apps can contain symbol information. With symbol information it is possible look up the function name that belongs to a part of the executable code of the app binary.
+
+There is a compiler flag that removes this symbol information from the binary. In Xcode this flag is DEPLOYMENT_POSTPROCESSING called. When set to Yes, the compiler will remove the symbol information, resulting in a smaller binary.
```

Comment:
Is this nature for all XCode versions? What if the stripping is there by default? Is it worth it to mention xcode version here if that's the case?

Reply:
The statements are true for all relevant Xcode versions. The DEPLOYMENT_POSTPROCESSING flag has not changed its default value over the years (and if it would, the readme still would be correct).

Reply:
I'll update the sentence though, it has a structural error 🙃